processor/attributesprocessor/config.go

```diff
@@ -18,14 +18,39 @@ import (
 	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
 )
 
-// Config specifies the set of attributes to be inserted, updated, upserted and deleted.
+// Config specifies the set of attributes to be inserted, updated, upserted and
+// deleted and the properties to include/exclude a span from being processed.
 // This processor handles all forms of modifications to attributes within a span.
+// Prior to any actions being applied, each span is compared against
+// the include properties and then the exclude properties if they are specified.
+// This determines if a span is to be processed or not.
 // The list of actions is applied in order specified in the configuration.
 type Config struct {
 	configmodels.ProcessorSettings `mapstructure:",squash"`
 
+	// Include specifies the set of span properties that must be present in order
+	// for this processor to apply to it.
+	// Note: If `exclude` is specified, the span is compared against those
+	// properties after the `include` properties.
+	// This is an optional field. If neither `include` and `exclude` are set, all spans
+	// are processed. If `include` is set and `exclude` isn't set, then all
+	// spans matching the properties in this structure are processed.
+	Include MatchProperties `mapstructure:"include"`
+
+	// Exclude specifies the set of span properties that must not be present in order
```

Comment:
>that must not be present

I have a feeling this does not convey the logic precisely enough. I can read it as "if the properties with specified keys do not exist in the span then processor will be applied to that span" which is obviously not what we do.

I would write it slightly differently, perhaps like this:

"This processor will not be applied to the Spans which match the specified properties"

or maybe there is a better English wording.

Reply:
That is much clearer thank you! I couldn't figure out how to phrase things anymore. I'll update it.